Brooks/Example/FileTests.hs

```diff
@@ -0,0 +1,38 @@
+module Example.FileTests (fileTests) where
+
+import New (PackageSym(..), RenderSym(..), PermanenceSym(..),
+  BodySym(..), BlockSym(..), StateTypeSym(..), 
+  StatementSym(..), ControlStatementSym(..), ValueSym(..), Selector(..),
+  FunctionSym(..), MethodSym(..), ModuleSym(..))
+import Prelude hiding (return,print,log,exp,sin,cos,tan)
+
+fileTests :: (PackageSym repr) => repr (Package repr)
+fileTests = packMods "FileTests" [fileDoc (buildModule "FileTests" [] [] [fileTestMethod] [])]
+
+fileTestMethod :: (RenderSym repr) => repr (Method repr)
+fileTestMethod = mainMethod "FileTests" (body [writeStory, block [readStory], 
+  goodBye])
+
+writeStory :: (RenderSym repr) => repr (Block repr)
+writeStory = block [
+  (varDecDef "e" int (litInt 5)),
```

Comment:
So `hlint` doesn't complain about these extra parens?

In any case, I would want to try to make this whole block "look like" code as much as possible.  For example, by inserting a blank line between the declarations and the executable code.